### PR TITLE
Update decidim to 0.22

### PR DESCRIPTION
**Pull request by anaghavl**
#### :tophat: What? Why?
Update decidim to 0.22.0

### CODE REVIEW

**Comment by mrcasals** [comment]
@anaghavl there are some conflicts, watch out!

File: spec/system/authorizations_spec.rb [hunk 1]
```diff
@@ -74,8 +74,6 @@ def fill_in_authorization_form
 
         within ".authorizations-list" do
           expect(page).to have_content("El padró")
-          expect(page).not_to have_link("El padró")
-          expect(page).to have_content(I18n.localize(authorization.granted_at, format: :long, locale: :ca))
```
Comment (mrcasals) [inline spec/system/authorizations_spec.rb:78]: Why was this changed? I think this should be kept 😕

Comment (anaghavl) [reply spec/system/authorizations_spec.rb:78]: They were both failing and I did a little digging to see it was removed in this PR in decidim, https://github.com/decidim/decidim/pull/2311/files

**Review by mrcasals — APPROVED** [review]
